packages/@uppy/transloadit/src/Client.js

```diff
@@ -126,6 +131,25 @@ export default class Client {
       .catch((err) => this.#reportError(err, { assembly, file, url, type: 'API_ERROR' }))
   }
 
+  /**
+   * Update the number of expected files in an already created assembly.
+   *
+   * @param {object} assembly
+   * @param {number} newNumberOfExpectedFiles
+   */
+  updateAssembly (assembly, tus_num_expected_upload_files) {
+    const url = new URL(assembly.assembly_ssl_url)
+    url.pathname = '/update_assemblies'
+    const body = JSON.stringify({
+      assembly_updates: [{
+        ...assembly,
+        tus_num_expected_upload_files,
+      }],
+    })
+    return this.#fetchJSON(url, { method: 'post', headers: this.#headers, body })
+      .catch((err) => this.#reportError(err, { url, type: 'API_ERROR' }))
```

Comment:
@tim-kos is this a reasonable design for the future API?

Reply:
I think we should rename it to `num_expected_upload_files`. This is also how we treat it on the API internally. No need to leak the implementation detail `tus` there I think. Otherwise looks good to me!